jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/importer/Importer.java

```diff
@@ -7,305 +7,408 @@
 import java.io.IOException;

```

Comment:
Thanks for all of the recommendations. Importer is definitely a good target for refactoring, I had a bit of trouble keeping it clean and working. Will make these changes on the refactor branch.